src/widget/city_overlay_risks.c

```diff
@@ -101,6 +101,15 @@ static int show_building_none(const building *b)
     return 0;
 }
 
+static int show_building_enemy(const building *b)
+{
+    return b->type == BUILDING_PREFECTURE
+        || b->type == BUILDING_WATCHTOWER || b->type == BUILDING_TOWER
+        || b->type == BUILDING_FORT || b->type == BUILDING_FORT_GROUND
+        || b->type == BUILDING_GATEHOUSE || b->type == BUILDING_PALISADE_GATE || b->type == BUILDING_PALISADE
+        || b->type == BUILDING_WALL; // The wall is not displayed! Why?
```

Comment:
The wall isn't an actual building.

Getting it to display will probably require a custom footprint/top drawing code, which I don't think is really worth the effort.